workshop: Update docs and support default cloud image

netoarmando opened the pull request:
Update instructions on how to build images starting with Fedora 34 using kickstart files used by Fedora to build its cloud images.

Change vagrant provisioning steps to support both prebuilt and default cloud images, removing the burden of maintaining boxes up-to-date, but also providing a way to build fresh images without external packer templates.

---
New Vagrant boxes available here: https://app.vagrantup.com/freeipa/boxes/freeipa-workshop/versions/20170517.01

flo-renaud reviewed (commented):
Hi @netoarmando 
thanks for the PR.
There are still references to the previous repo in the curriculum (for instance README.rst mentions `$ git clone https://github.com/freeipa/freeipa-workshop.git` but the workshop was moved to https://github.com/freeipa/freeipa/ in doc/workshop).

README.rst also instructs `$ vagrant box add netoarmando/freeipa-workshop`, this should be replaced with fedora/33-cloud-base.

README.facilitators also mentions the wrong box.

rcritten commented:
Would this affect someone doing a workshop at something like a conference with limited bandwidth? I think it would mean more packages need to be retrieved.

netoarmando commented:
@rcritten I changed this to support both cases, using prebuilt images as default.
@flo-renaud I've updated the documentation to fix your comments and reflect the new changes.

flo-renaud commented on doc/workshop/building.rst:
```diff
@@ -8,55 +8,43 @@ Requirements
+- Clone the repo and checkout latest Fedora release branch::
 
+  $ git clone https://pagure.io/fedora-kickstarts.git
+  $ git checkout f34
```
The step "cd fedora-kickstarts" is missing before git checkout

netoarmando replied:
Fixed.

flo-renaud commented on doc/workshop/building.rst:
```diff
@@ -8,55 +8,43 @@ Requirements
-  mkdir box && cd box && tar -xf ../fedora-28-x86_64-libvirt.box
-  virt-sparsify --in-place box.img
-  tar -czf ../fedora-28-x86_64-libvirt.box * && cd .. && rm -rf box
+  $BIN_PACKER build packer-template-fedora.json
```
"cd $FREEIPA_REPO/doc/workshop" is missing

netoarmando replied:
Fixed.

flo-renaud reviewed (commented):
Hi @netoarmando 
please find additional inline comments

flo-renaud commented:
Re-running `test_authconfig` as the test timed out
I had a look at the latest changes, LGTM. ACK for when PRCI completes

flo-renaud commented:
master:

* 584952babffb62346802d43df8bb46ed68897f15 workshop: Update docs and support default cloud image
